share/src/main/resources/META-INF/components/documentlibrary/onlyoffice-doclib-multi-download-as.js

```diff
@@ -44,23 +44,12 @@
                 fn: function (response) {
                     waitDialog.destroy()
 
-                    var form = document.createElement("form");
-                    form.method = "GET";
-                    form.action = Alfresco.constants.PROXY_URI + response.json.downloadUrl;
-                    document.body.appendChild(form);
-
-                    var d = form.ownerDocument;
-                    var iframe = d.createElement("iframe");
-                    iframe.style.display = "none";
-                    YAHOO.util.Dom.generateId(iframe, "downloadArchive");
-                    iframe.name = iframe.id;
-                    document.body.appendChild(iframe);
-
```

Comment:
You can leave the current code. Just need to add parameter "a"=true. Like this:
                                var input = document.createElement("input");
                                input.type = "hidden"
                                input.name = "a"
                                input.value = "true";
                                form.appendChild(input);